SU-HSDO/suhumsci
# SHS-5904: Add new "log out" block to existing sites

# READY FOR REVIEW

## Summary
Adds new logout button to existing HumSci sites on both HumSci Colorful and HumSci Traditional themes.

## Need Review By (Date)
2024-12-11

## Urgency
medium

## Steps to Test
1. Login to the Colorful site: https://hs-colorful-1fwsletf5ghnmd1kmk0ngkockhhydj2r.tugboatqa.com
2. Verify that there is a logout button in the footer

![Screenshot 2024-12-10 at 10 31 50 AM](https://github.com/user-attachments/assets/bb35393f-cbc1-43e7-a0f5-519b40f110e3)


4. Login to the Traditional site: https://hs-traditional-1fwsletf5ghnmd1kmk0ngkockhhydj2r.tugboatqa.com
5. Verify that there is a logout button in the footer

![Screenshot 2024-12-10 at 10 32 29 AM](https://github.com/user-attachments/assets/cc21836d-2c14-42de-a606-98938e94a169)


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208977450116179
